package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "template-feo-react-babel",
+  "version": "1.0.0",
+  "description": "Group 3",
+  "main": "index.html",
+  "scripts": {
+    "start": "nodemon -e js,graphql,env -w server -w .env  -w . server/server.js"
+  },
+  "dependencies": {
+    "body-parser": "^1.19.0",
+    "bulma": "^0.9.3",
+    "dotenv": "^10.0.0",
+    "express": "^4.17.1",
+    "morgan": "^1.10.0",
+    "nodemon": "^2.0.7",
+    "request": "^2.88.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/.git"
+  },
+  "keywords": [
+    "Bulma",
+    "Fetch"
+  ],
+  "author": "Group 3",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/jsharples777/template-feo-react-babel/issues"
+  },
+  "homepage": "https://github.com/jsharples777/template-feo-react-babel#readme"
+}
```

Comment:
You'll probably have to take us through a few of these Jim, as you mentioned the other night